addon/templates/components/frost-object-browser-inline.hbs

```diff
@@ -24,7 +24,11 @@
     {{/if}}
   <div class="content">
     <div class="tools">
-      <div class="sort"/>
+      <div class="sort">
+        {{#if sortData}}
+          {{frost-sort sortableProperties=sortData.sortableProperties sortParams=sortData.sortParams onChange=onSortChange}}
```

Comment:
I've been getting into the habit of grouping data related to sub-components (this is helpful when using redux).

I also wanted to avoid adding more attributes to the API than necessary, but I don't mind changing it if others would rather have two separate attributes.